### PR TITLE
fix(logging): add error log when getting services from Docker server

### PR DESCRIPTION
**Pull request by she11sh0cked**
## Proposed change

This PR addresses an issue where a server fails to retrieve services from Docker without raising an error, potentially causing confusion regarding server functionality. The fix adds an error log to provide clarity in such scenarios.

## Type of change

- [ ] New service widget
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation only
- [ ] Other (please explain)

## Checklist:

- [ ] If applicable, I have added corresponding documentation changes.
- [ ] If applicable, I have reviewed the [feature](https://gethomepage.dev/latest/more/development/#new-feature-guidelines) and / or [service widget guidelines](https://gethomepage.dev/latest/more/development/#service-widget-guidelines).
- [x] I have checked that all code style checks pass using [pre-commit hooks](https://gethomepage.dev/latest/more/development/#code-formatting-with-pre-commit-hooks) and [linting checks](https://gethomepage.dev/latest/more/development/#code-linting).
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.

### CODE REVIEW

**Comment by she11sh0cked** [comment]
See the Discord [discussion](https://discord.com/channels/1019316731635834932/1219675365711286382) for details.

TLDR: My homepage failed to resolve any Docker services. After some debugging I found out that a silent error was thrown, because homepage couldn't access the secrets.

image
image

**Review by shamoon — APPROVED** [review]
Thats fine. I cant remember if there isnt logging there for a reason but seems reasonable thanks
